### PR TITLE
ENH add pzflow

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge/staged-recipes This `noarch: python` recipe is ready for review!

File: recipes/pzflow/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - poetry
```
Comment (BastianZim) [inline recipes/pzflow/meta.yaml:21]:
```suggestion
    - poetry-core >=1.0.0
```

File: recipes/pzflow/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    # I don't think any of these constraints are real but here I am
+    - python >=3.8
+    - jax =0.4.2
+    - optax =0.1.4
+    - pandas >=1.1
+    - dill =0.3.6
+    - tqdm =4.64.1
+
+test:
```
Comment (BastianZim) [inline recipes/pzflow/meta.yaml:31]: Does pip check not work?

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/pzflow/meta.yaml (hunk 2 above)
Comment (beckermr) [reply recipes/pzflow/meta.yaml:31]: let's test it!

Comment (beckermr) [reply recipes/pzflow/meta.yaml:31]: I ignore pip check out of habit. When it passes, that's fine. When it fails, we ignore it. So why bother?

Comment (beckermr) [reply recipes/pzflow/meta.yaml:31]: I added it to see if it passes here.

Comment (BastianZim) [reply recipes/pzflow/meta.yaml:31]: I guess, yeah. I find it helpful to see if something fails down the line. Most of the time, it's nothing we can control, but sometimes it's a regression in a feedstock, and then it's a nice trip wire.

**Comment by beckermr** [comment]
`pip check` appears to work! Back to you @BastianZim!

**Comment by beckermr** [comment]
Thank you!

**Pull request merged**